### PR TITLE
Cleaned up unbindEvent_ calls

### PR DESCRIPTION
**Pull request by BeksOmega**
## The basics

- [X] I branched from develop
- [X] My pull request is against develop
- [X] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

N/A

### Proposed Changes

Every time there was an `unbindEvent_` call that wasn't part of disposing of the entire object I set the event data object to null. This pattern was already used some places, like scrollbars, and the workspace comment renderer.

### Reason for Changes

This makes sure that if the function which contains the unbinding is called, we don't have unnecessary calls to `unbindEvent_`.

If we are disposing of the entire object, setting the event data object to null is unnecessary because we'll be getting rid of that reference anyway.

### Test Coverage

These changes shouldn't affect behavior.

### Documentation

N/A

### Additional Information

N/A

### CODE REVIEW

File: core/field_textinput.js [hunk 1]
```diff
@@ -362,9 +362,11 @@ Blockly.FieldTextInput.prototype.bindInputEvents_ = function(htmlInput) {
...
   if (this.onKeyDownWrapper_) {
     Blockly.unbindEvent_(this.onKeyDownWrapper_);
+    this.onKeyDownWrapper_ = null;
   }
   if (this.onKeyInputWrapper_) {
     Blockly.unbindEvent_(this.onKeyInputWrapper_);
+    this.onKeyInputWrapper_ = null;
   }
   if (this.onBlurInputWrapper_) {
     Blockly.unbindEvent_(this.onBlurInputWrapper_);
```
Comment (samelhusseini) [inline core/field_textinput.js]: Missed one?

Comment (samelhusseini) [reply core/field_textinput.js]: ``onBlurInputWrapper_``

File: core/workspace_comment_svg.js [hunk 2]
```diff
@@ -49,6 +49,21 @@ goog.require('Blockly.WorkspaceComment');
...
+   */
+  this.onMouseUpWrapper_ = null;
+
+  /**
+   * Mouse move event data.
+   * @type {?Blockly.EventData}
+   * @private
+   */
+  this.onMouseMoveWrapper_ = null;
+
```
Comment (samelhusseini) [inline core/workspace_comment_svg.js]: Looks like these were just added, but not set to anything.

File: core/field_textinput.js (hunk 1 above)
Comment (BeksOmega) [reply core/field_textinput.js]: Haha I think this may have appeared when I rebased, will fix!

File: core/workspace_comment_svg.js (hunk 2 above)
Comment (BeksOmega) [reply core/workspace_comment_svg.js]: They're [used inside the workspace_comment_render_svg](https://github.com/google/blockly/blob/ba6dfd812552e0823ed75f3dc5133a267cc6fbc9/core/workspace_comment_render_svg.js#L266) file, so I believe that means they need to be declared here. Honestly I'm not sure though.

Comment (samelhusseini) [reply core/workspace_comment_svg.js]: Sounds good.

**Pull request merged**